src/test/rustdoc-ui/automatic-links.rs

```diff
@@ -0,0 +1,60 @@
+#![deny(url_improvements)]
```

Comment:
Can you rename the file to match the name of the lint?

Reply:
:+1: